pkg/cluster/task/systemd.go

```diff
@@ -24,6 +25,7 @@ type SystemCtl struct {
 	host   string
 	unit   string
 	action string
+	reload bool
```

Comment:
just set `action` to `reload`

Reply:
It means `daemon-reload`.

Reply:
Um, then it'd better be named as `reloadDaemon`